lib/modules/smtp/mail_honeypot/files/smtp_honeypot.py

```diff
@@ -0,0 +1,333 @@
+
+import socket
+import sys
+import errno
+import time
+import threading
+import datetime
+import asyncore
+import asynchat
+import os
+import json
+
+
+MAILSERVER_NAME= 'OWASP Python Honeypot'
```

Comment:
shouldn't this be in module configuration?

Reply:
yes , but then how can I pass from module_config to this script running in the container? 
one option could be make an another conf file put it in the container and in the script read from that file.
any other way? @Ali-Razmjoo

Reply:
@dhirensr environment variable maybe?

Reply:
@Ali-Razmjoo : thanks for the hint/suggestion. :+1: